src/index.tsx

```diff
@@ -4,6 +4,11 @@ import './index.css';
 import * as serviceWorker from './serviceWorker';
 import { Router } from './Router';
 
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Is this guarenteed?

Reply:
yes. 
https://create-react-app.dev/docs/adding-custom-environment-variables/